integrations/pytorch/export.py

```diff
@@ -0,0 +1,206 @@
+# Copyright (c) 2021 - present / Neuralmagic, Inc. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""
+#####
+Command help:
+usage: export.py [-h] --arch-key ARCH_KEY [--pretrained PRETRAINED]
```

Comment:
are those up-to-date? Not sure if the argparser changed much because of the split but might be worth double-checking?

Reply:
Yes, those are up to date, generated fresh using --help